pages/bibliography.md

```diff
@@ -1,11 +1,48 @@
+[Home](https://michaelrappa.github.io/usa-migration/) |
+[Analysis](https://michaelrappa.github.io/usa-migration/pages/analysis.html) |
+[Background](https://michaelrappa.github.io/usa-migration/pages/background.html) |
+[Reflections](https://michaelrappa.github.io/usa-migration/pages/reflections.html) |
+[Workflow](https://michaelrappa.github.io/usa-migration/pages/workflow.html) |
+[Credits](https://michaelrappa.github.io/usa-migration/pages/credits.html) |
+[Bibliography](https://michaelrappa.github.io/usa-migration/pages/bibliography.html) |
+[Sources](https://michaelrappa.github.io/usa-migration/pages/sources.html)
+
+
+
 # Bibliography
 
----
+1.
+Galka, Max. 2016. "Here's Everyone Who's Immigrated to the U.S. Since 1820." May 3. http://metrocosm.com/animated-immigration-map/.
```

Comment:
```suggestion
Galka, Max. 2016. "Here's Everyone Who's Immigrated to the U.S. Since 1820." May 3. http://metrocosm.com/animated-immigration-map/.
```

Reply:
You should make the URLs in your bibliography into links! That way, the reader can easily click them.